spec/helpers/category_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+RSpec.describe CategoryHelper , type: :helper do
+  let!(:category_1) { create(:category, name: "name 1",enabled: false) }
+  let!(:category_2) { create(:category, name: "name 2",enabled: true) }
+  let!(:category_3) { create(:category, name: "name 3",enabled: false) }
+  let!(:category_4) { create(:category, name: "name 4",enabled: true) }
```

Comment:
Should have white space before `enabled `.
Please do an update that for four categories!